### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HassanatOladini**
Additional information added

### CODE REVIEW

**Comment by HassanatOladini** [comment]
@tanyapowell

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # hello-world
+Just practicing
```
Comment (tanyapowell) [inline README.md]: In markdown you should add a blank line after a heading

Comment (HassanatOladini) [reply README.md]: Ok, noted!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
 # hello-world
```
Comment (tanyapowell) [inline README.md]: Can you change the title to be `Hello World` instead of `hello-world`?

**Review by tanyapowell — APPROVED** [review]
💃🏾

**Pull request closed without merging**